### PR TITLE
Tdrd 5 content changes

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: app/views/draftmetadata/draftMetadataChecks.scala.html [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
+                <p class="govuk-body">Please wait while we check your metadata against the uploaded records. This may take a few minutes.</p>
                 <div class="govuk-notification-banner" id="draft-metadata-checks-completed-banner" tabindex="-1" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" hidden>
                     <div class="govuk-notification-banner__header">
                         <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
                         Important
                         </h2>
                     </div>
                     <div class="govuk-notification-banner__content">
-                        <p class="govuk-notification-banner__heading">Your CSV has been checked.</p>
+                        <p class="govuk-notification-banner__heading">Your metadata has been checked.</p>
```
Comment (TomJKing) [inline app/views/draftmetadata/draftMetadataChecks.scala.html:23]: According to the prototype HTML this should be an `h3` tag: 
```
<h3 class="govuk-notification-banner__heading">
      Your metadata has been checked.
</h3>
```

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor comment. Thanks

File: app/views/draftmetadata/draftMetadataChecks.scala.html (hunk 1 above)
Comment (vimleshtna) [reply app/views/draftmetadata/draftMetadataChecks.scala.html]: Good catch. Updated it, thanks.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**